### PR TITLE
Tweak note about 'IDs' being keyword to also mention 'codes'

### PR DESCRIPTION
**Pull request by webmat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -503,11 +503,12 @@ of a breaking change with these two widely used fields in Beats.
 Any future field that will be indexed for full text search in ECS will however
 follow the multi-field convention where `text` indexing is nested in the multi-field.
 
-### IDs are keywords not integers
+### IDs and codes are keywords, not integers
```
Comment (ruflin) [inline README.md]: Are you thinking of some specific fields here?

Comment (webmat) [reply README.md]: Yes, error codes. Came up during a discussion with @graphaelli over on #154

**Comment by ruflin** [comment]
The reason I'm hesitant to merge this is because there are some instances like status_code where things are long and as I see this as a good thing. Perhaps it can be rephrased to "in most cases".

**Comment by webmat** [comment]
Gotcha. I'll tweak accordingly.

**Comment by webmat** [comment]
Reworded. Although I find this a bit wordy.

**Comment by webmat** [comment]
I agree README is getting verbose. But my plan is definitely to split off some of these discussions into secondary documentation files, to streamline the readme again.
